apps/src/dance/blockly/blockDefinitions/when_run.ts

```diff
@@ -7,7 +7,7 @@ import {
 import {commonI18n} from '@cdo/apps/types/locale';
 
 const definition: BlockJson = {
-  type: 'when_run',
+  type: 'Dancelab_whenRun',
```

Comment:
@mikeharv just an FYI: I noticed when pulling in the music lab blocks, we had two competing definitions for `when_run` causing issues depending on the order in which they were installed. I opted to rename this one with `Dancelab` prefix because it does do something dance-specific stuff (calling `whenSetup()`). I think there's a case to be made for also adding a similar Music Lab prefix to the Music Lab blocks since it also has some music lab-specific code in the handler, but that's something we could look into later.

There might be other blocks where this is the case though; for example, I think the procedure blocks may have a similar issue.